src/structures.rs

```diff
@@ -23,13 +23,14 @@ pub struct SvPackageDeclaration {
 #[derive(Debug, Serialize, Clone)]
 pub struct SvParameter {
     pub identifier: String,
-    pub value: Option<String>,
+    pub expression: Option<String>,
     pub paramtype: SvParamType,
     pub datatype: Option<SvDataType>,
-    pub datatype_status: SvParamStatus,
+    pub datatype_overridable: bool,
     pub classid: Option<String>,
     pub signedness: Option<SvSignedness>,
-    pub signedness_status: SvParamStatus,
+    pub signedness_overridable: bool,
+    pub bit_num: Option<u64>,
```

Comment:
Would it make sense to rename? bit_num -> num_bits
Perhaps I'm mis-understanding the purpose.

Reply:
Fixed.